From 8f9a102eb54a10074b0593efde582c26bc3686b6 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Thu, 15 Dec 2022 09:31:35 +0530
Subject: [PATCH] Dont't add TvSettings to category Home

When TvSettings is added to category home with TvSampleLeanbackLauncher
then it causes conflict between the 2 and stops Android boot.

Tracked-On: OAM-104028
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 Settings/AndroidManifest.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Settings/AndroidManifest.xml b/Settings/AndroidManifest.xml
index 63cdf7207..53488e873 100644
--- a/Settings/AndroidManifest.xml
+++ b/Settings/AndroidManifest.xml
@@ -693,7 +693,7 @@
              android:theme="@style/FallbackHome">
             <intent-filter android:priority="-1000">
                 <action android:name="android.intent.action.MAIN"/>
-                <category android:name="android.intent.category.HOME"/>
+                <!--category android:name="android.intent.category.HOME"/-->
                 <category android:name="android.intent.category.DEFAULT"/>
             </intent-filter>
         </activity>
-- 
2.17.1

